CONTRIBUTING.md

```diff
@@ -14,29 +14,41 @@ Docker does things, you'll feel right at home.
 Otherwise, please read [Docker's contributions
 guidelines](https://github.com/docker/docker/blob/master/CONTRIBUTING.md).
 
-# Building using docker
+# Building
 
 The requirements to build Machine are:
 
-1. A running instance of Docker
+1. A running instance of Docker or a Golang 1.5 development environment
 2. The `bash` shell
 3. [Make](https://www.gnu.org/software/make/)
 
-Call `export USE_CONTAINER=true` to instruct the build system to use containers to build.
+## Build using Docker containers
 
-# Alternative: build using go only
+To build the ``docker-machine`` binary using containers, simply run:
 
-Alternatively, you can build without docker, using only golang.
-
-[Install and setup go](https://golang.org/doc/install), then clone the machine repository inside your gopath.
+    $ export USE_CONTAINER=true
+    $ make build
 
-## Building
+## Local Go 1.5 development environment
+
+Make sure the source code directory is under a correct directory structure to use Go 1.5 vendoring;
+example of cloning and preparing the correct environment ``GOPATH``:
+````
+    mkdir -p projects/docker-machine/src/github.com/docker
+    export GOPATH="$PWD/projects/docker-machine"
+    cd projects/docker-machine/src/github.com/docker
+    git clone https://github.com/docker/machine
+    cd machine
+    export GOPATH="$GOPATH:$PWD"
```

Comment:
Well, the imports from this same project (`docker/machine`) are not vendor'ed (obviously) and not in `GOPATH`, I remember trying without this line and getting errors.